api/rql/internal/primary/meta/testdata/README.md

```diff
@@ -0,0 +1,21 @@
+`before_munging.json` was generated by running the following type through the [jsonschema](https://github.com/alecthomas/jsonschema) library (specifically ekinanp's fork):
```

Comment:
Are you working on upstreaming these?

Reply:
Looks like schema_test.go mentions the issue created to start that process.

Reply:
Yeah, haven't gotten around to it.